src/cryptography/hazmat/primitives/padding.py

```diff
@@ -122,3 +126,53 @@ def finalize(self):
         res = self._buffer[:-pad_size]
         self._buffer = None
         return res
+
+
+class PKCS7(_BytePadding):
```

Comment:
I don't get the point. I understood before that it was not clean to inherit from  PKCS7 to create ANSIX923. But here I moved into a private class the common code for any padding based on byte. For example, it could use to also implement ISO 10126 or ISO/IEC 7816-4.
If I don't use subclass, I will have to duplicate most of the code.